extensions/vtk/src/utils/setMPRLayout.js

```diff
@@ -42,3 +53,65 @@ export default function setMPRLayout(displaySet) {
     );
   });
 }
+
+/*
```

Comment:
Just remove this instead of leaving a massive comment block

Reply:
Oops!